tests/e2e/vsan_stretched_cluster.go

```diff
@@ -3450,13 +3588,19 @@ var _ = ginkgo.Describe("[vsan-stretch-vanilla] vsan stretched cluster tests", f
 			}
 		}
 
+		if rwxAccessMode {
+			command = execRWXCommandPod
+		} else {
+			command = execCommand
+		}
+
 		for i := 0; i < 2; i++ {
-			pod, err := createPod(ctx, client, namespace, nil,
-				[]*v1.PersistentVolumeClaim{pvclaimsWithDelete[i]}, false, execCommand)
-			framework.Logf("Created pod %s", pod.Name)
+			pod, err = createPod(ctx, client, namespace, nil, []*v1.PersistentVolumeClaim{pvclaimsWithDelete[i]}, false, command)
 			gomega.Expect(err).NotTo(gomega.HaveOccurred())
+			framework.Logf("Created pod %s", pod.Name)
 			pods = append(pods, pod)
 		}
+
 		framework.Logf("Stopping vsan-health on the vCenter host")
 		vcAddress := e2eVSphere.Config.Global.VCenterHostname + ":" + sshdPort
 		err = invokeVCenterServiceControl(ctx, stopOperation, vsanhealthServiceName, vcAddress)
```

Comment:
check all services up and running state in after each block

Reply:
Yes, it is getting verified in after Each